Source/WebKit/UIProcess/API/ios/WKWebViewIOS.mm

```diff
@@ -3074,8 +3084,10 @@ - (void)_endLiveResize
     [self _didStopDeferringGeometryUpdates];
 
     [self _doAfterNextPresentationUpdate:^{
-        [liveResizeSnapshotView removeFromSuperview];
-    }];    
+        [self _doAfterVideoFullscreenPlayerLayerBoundsHaveChanged:^{
```

Comment:
What if there is no AV player after the live resize? Should this immediately call the callback in this case?

Reply:
yes, good point; it should call the callback in that case. Where would a good place to do this be?

Reply:
Probably either in `_doAfterVideoFullscreenPlayerLayerBoundsHaveChanged` or if there is some way to check if an AV player is playing currently and will be resized, I am not sure off hand how to do that.

Also what if the window does a live resize, ends up at the same size, and then the AV player doesn't need resizing? We still want to call this callback.